FitTracker/app/src/main/java/com/pps/globant/fittracker/mvp/presenter/LoginPresenter.java

```diff
@@ -11,7 +15,31 @@
     public LoginPresenter(LoginModel model, LoginView view) {
         this.model = model;
         this.view = view;
-        view.setLabel("Hi there");
     }
 
+    @Subscribe
+    public void onFetchingFbUserDataCompletedEvent(FacebookLoginProvider.FetchingFbUserDataCompletedEvent event) {
+        view.setLabelFb("name: " + event.fbUser.getName());
```

Comment:
Use string format for that. See https://developer.android.com/reference/java/util/Formatter